aspnetcore/blazor/hosting-models.md

```diff
@@ -29,7 +29,7 @@ To create a Blazor app using the client-side hosting model, use the **Blazor Web
 
 After selecting the **Blazor WebAssembly App** template, you have the option of configuring the app to use an ASP.NET Core backend by selecting the **ASP.NET Core hosted** check box ([dotnet new blazorwasm --hosted](/dotnet/core/tools/dotnet-new)). The ASP.NET Core app serves the Blazor app to clients. The Blazor WebAssembly app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction).
 
-The templates include the *blazor.webassembly.js* script that handles:
+The templates include the `blazor.webassembly.js` script that handles:
```

Comment:
Have we switched to using code formatting for file names and paths? I thought someone told me at some point to use italics, but if this is the new guidance then that's fine.

Reply:
No, italics is still what we want. **_BUT_** if we do that here, it will become the *dazzling.webassembly.js* file :smile: **in French** (for example). Therefore, we'll have to code fence these until the docs engineers work further on the no-loc system.